### PR TITLE
{chem}[foss/2017a] propy v1.0

### PR DESCRIPTION
**Pull request by dplacencia**
Propy is a protein description software very usefull. He has been used sucessfully in Universidad Central Marta Abreu de Las Villas (UCLV) .

### CODE REVIEW

**Comment by dplacencia** [comment]
Thank you very much for the attachments. I will make the changes as soon as possible.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/fc966239448bfc8418e235e53c86eda6 for a full test report.

File: easybuild/easyconfigs/p/propy/propy-1.0-foss-2017a-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+version = '1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = ""
+description = """ Propy is a protein description software. He allows to analyze Sequence-derive
+ structural and physicochemical features, which are very useful to representing
+ and distinguishing proteins or peptides of different structural, functional and
+ interaction properties, which have been widely used in developing methods and 
+ software for predicting protein structural and functional classes, protein-protein
+ interactions, drug-target interactions, protein substrates, among others... """
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/p/propy/propy-1.0-foss-2017a-Python-2.7.13.eb]: suggestion:
```python
description = """Propy is a protein description software. It allows analyzing sequence-derived structural and 
 physicochemical features, which are very useful in representing and distinguishing proteins or peptides of
 different structural, functional and interaction properties. These have been widely used in developing methods
 and software for predicting protein structural and functional classes, protein-protein interactions, drug-target
 interactions, protein substrates, among others."""
```

**Comment by migueldiascosta** [comment]
Going in, thanks @dplacencia!

**Pull request merged**